src/lists/list-item.jsx

```diff
@@ -507,6 +507,7 @@ const ListItem = React.createClass({
       },
 
       primaryText: {
+        order:1,
```

Comment:
The secondary text is rendered in the DOM before the primary text. Hence I am using the order to align them in order. I also removed the floats.